projects/Rockchip/options

```diff
@@ -76,12 +76,12 @@
     SYSTEM_PART_START=32768
 
   # build and install driver addons (yes / no)
-    DRIVER_ADDONS_SUPPORT="yes"
+    DRIVER_ADDONS_SUPPORT="n"
```

Comment:
Should be `no`

Reply:
Setting this to no and trying to upgrade an install that had this set to yes caused issues with kodi not starting due to missing installed addon several months ago, I think it will be safer to just remove the addons from DRIVERS_ADDONS

Reply:
ups that should be yes, fat fingers i guess :(

fixed